content/en/security/threats/_index.md

```diff
@@ -45,6 +45,8 @@ Investigate and triage security signals in the [Signals Explorer][8]. View detai
 
 {{< img src="security/cws/signals_explorer.png" alt="CSM Signals Explorer page" width="100%">}}
 
+<div class="alert alert-info">Datadog is introducing a new feature called Active Protection to address the threats detected in your environment automatically. Active Protection is in private beta. Fill out this <a href="https://docs.google.com/forms/d/e/1FAIpQLSfzQARsTPr3tiJDnS_4bGx7w35LDfAbGUggaUzHYoL0dIUMWQ/viewform">form</a> to request access.</div>
```

Comment:
Heads up, we also have this style where you can make the form accessible through a button. Up to you what you prefer. It would looks like this:

```
{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfzQARsTPr3tiJDnS_4bGx7w35LDfAbGUggaUzHYoL0dIUMWQ/viewform" btn_hidden="false" header="Join the Beta!">}}
Datadog is introducing a new feature called Active Protection to address the threats detected in your environment automatically. Active Protection is in private beta. Fill out the form to request access.
{{< /callout >}}
```

![image](https://github.com/DataDog/documentation/assets/84536271/2ba87078-659e-4078-9cb9-00a3c4856d27)

Reply:
Can you help merge this PR? 🙏